src/vs/workbench/contrib/interactive/browser/interactiveEditor.ts

```diff
@@ -180,10 +185,13 @@ export class InteractiveEditor extends EditorPane {
 			const contentHeight = this.#codeEditorWidget.getContentHeight();
 
 			if (this.#dimension) {
-				this.#notebookWidget.value!.layout(this.#dimension.with(this.#dimension.width, this.#dimension.height - contentHeight), this.#rootElement);
+				this.#notebookWidget.value!.layout(this.#dimension.with(this.#dimension.width, this.#dimension.height - contentHeight - 2 * INPUT_CONTAINER_PADDING), this.#rootElement);
 				this.#codeEditorWidget.layout(new DOM.Dimension(this.#dimension.width, contentHeight));
-				this.#inputEditorContainer.style.top = `${this.#dimension.height - contentHeight}px`;
+				this.#inputEditorContainer.style.top = `${this.#dimension.height - contentHeight - 2 * INPUT_CONTAINER_PADDING}px`;
 				this.#inputEditorContainer.style.width = `${this.#dimension.width}px`;
+				this.#inputEditorContainer.style.height = `${contentHeight}px`;
+				this.#inputEditorContainer.style.paddingTop = `${INPUT_CONTAINER_PADDING}px`;
+				this.#inputEditorContainer.style.paddingBottom = `${INPUT_CONTAINER_PADDING}px`;
```

Comment:
This block of code is really, really similar to the above.  Any way these can be refactor into one function?